src/app/component/places/places.component.ts

```diff
@@ -0,0 +1,178 @@
+import { TranslateService } from '@ngx-translate/core';
+import { Component, OnInit, ViewChild, DoCheck } from '@angular/core';
+import { LocalStorageService } from '@global-service/localstorage/local-storage.service';
+import { HttpClient } from '@angular/common/http';
+import { MatDrawer } from '@angular/material';
+import { AgmMarker } from '@agm/core';
+
+import { markers } from './Data.js';
+import { cards } from './Data.js';
```

Comment:
should be import { cards, markers  } from './Data.js';

Reply:
DONE ✔️